src/thing/remove.test.ts

```diff
@@ -2290,15 +2603,38 @@ describe("removeLiteral", () => {
     });
   });
 
+  it("accepts time as Literal", () => {
+    const thingWithTime = getMockThingWithLiteralFor(
+      "https://some.vocab/predicate",
+      "13:37:42Z",
+      "time"
+    );
+
+    const updatedThing = removeLiteral(
+      thingWithTime,
+      "https://some.vocab/predicate",
+      DataFactory.literal(
+        "13:37:42Z",
+        DataFactory.namedNode("http://www.w3.org/2001/XMLSchema#time")
+      )
+    );
+
+    expect(
+      updatedThing.predicates["https://some.vocab/predicate"].literals
+    ).toStrictEqual({
+      "http://www.w3.org/2001/XMLSchema#time": [],
+    });
+  });
+
```

Comment:
I think this is in the wrong `describe` block?

Reply:
I believe this is needed. We do the same check for DateTime, seeing if removeLiteral accepts that data type, so I think it makes sense to do this for time too??

Reply:
Ah sorry! I thought this was in the tests for Datetime, but it's actually in the block for `removeLiteral`. Never mind then!